io.github.quodlibet.ExFalso.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 app-id: io.github.quodlibet.ExFalso
 runtime: org.gnome.Platform
-runtime-version: '40'
+runtime-version: '41'
```

Comment:
42 is out.

Reply:
Thanks, 42 crashes currently due to two libsoup versions being loaded.. Needs to be investigated